spec/unit/reports/servicenow_spec.rb

```diff
@@ -245,6 +245,7 @@
         it 'decrypts the password' do
           expected_incident = {
             short_description: short_description_regex('failed'),
+            description: 'This incident was created based on the following conditions: failures. See the PE console for the full report. You can access the PE console at test_console.',
```

Comment:
Let's make this a regex like `short_description` where the regex contains the 'high-level' stuff (so the satisfied conditions and maybe PE console url?). That way unit tests still test essential behavior while making them less of a PITA to maintain if we change something simple like e.g. the wording.

Reply:
Makes sense, updated!